packages/stream_chat_flutter/lib/src/message_input/stream_message_input.dart

```diff
@@ -851,6 +929,11 @@ class StreamMessageInputState extends State<StreamMessageInput>
   /// this will open the native file system and allow the user to select one
   /// or more files.
   Future<void> _onAttachmentButtonPressed() async {
+    _effectiveController.addListener(
+      () {
+        setState(() {});
+      },
+    );
```

Comment:
Doubt:- Is there any reason? why we are refreshing the state to each event call?  I

Reply:
Not using this function, removing the code.

Reply:
Removed in UA-621.